### DISCUSSION TITLE
Fill in the field with @updateAt only on update

### DISCUSSION THREAD

**Discussion by leoShabalkin** [prisma/prisma#6557] [Q&A] [answered]
I have the following schema
```prisma
model Data {
     id          Int @id       @default(autoincrement())`
     created DateTime    @default(now())`
     updated DateTime? @updateAt`
}
```
when executing a request 

`prisma.model.create(...)` the value current date is put in the field updated. How to make a value in a field only appear during an operation update(`prisma.model.update(...)`)? 
Of course, I can make a request like
`prisma.model.create({data: {..., updated: null})`, but it looks like a hack

**Comment by ryands17** [comment, marked as answer]
@leoShabalkin 👋 

This is normal behaviour for ORM's where the `updatedAt` is set at the `createdAt`. Could you explain your use case for this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
